tests/context-info.sh

```diff
@@ -400,7 +400,12 @@ run_test "Wrong base64 format" \
          "def_bad_b64.txt" \
          -m "ERROR" \
          -u "The length of the base64 code found should be a multiple of 4" \
-         -n "bytes left to analyze from context"
+         -n "bytes left to analyze from context" \
+
+run_test "EOF base64 format" \
+         "def_bad_b64_eof.txt" \
+         -m "Finished. No valid base64 code found" \
+         -n "ERROR" \
```

Comment:
There shouldn't be a trailing backslash here since there is no continuation line.